internal/graph/check.go

```diff
@@ -250,12 +257,15 @@ func (cc *ConcurrentChecker) checkDirect(ctx context.Context, crc currentRequest
 	// Convert the subjects into batched requests.
 	toDispatch := make([]directDispatch, 0, subjectsToDispatch.Len())
 	subjectsToDispatch.ForEachType(func(rr *core.RelationReference, resourceIds []string) {
+		chunkCount := 0.0
```

Comment:
would it perhaps be easier to analyze the graphs if instead of "observing chunks" we observe the actual length of the chunk, so we don't have to do the mental math based on the default chunk size (which may as well become configurable)? Also with the current histogram bucket granularity, chances are everything falls within 1 or 2, which wouldn't offer a lot of information. If we decide to move to actual chunk length, perhaps we could use a different bucket size progression like `1, 5, 10, 20, 40, 80, 160, 320, 640, 1280...`

Reply:
after chatting with Joey, I've reconsidered my stand here, since it's more useful to see "how much we are dispatching" versus the size of the dispatched requests